rclcpp/include/rclcpp/executor.hpp

```diff
@@ -343,6 +351,10 @@ class Executor
    * a single-thread model of execution.
    * Adding subscriptions, timers, services, etc. with blocking callbacks will cause this function
    * to block (which may have unintended consequences).
+   *
+   * It tries to collect and execute a single work just only once that is available in any entities
+   * within a timeout.
+   *
```

Comment:
Again, this is just a reiteration of the brief description, so I wouldn't add this.